src/Command/CodeStudio/CodeStudioWizardCommand.php

```diff
@@ -161,8 +161,10 @@ private function createProjectAccessToken(array $project, string $projectAccessT
       $this->checklist->completePreviousItem();
     }
     $this->checklist->addItem("Creating access token named <comment>$projectAccessTokenName</comment>");
+    $dateTime = new \DateTime('tomorrow');
```

Comment:
This token needs to be long-lived since it's going to be stored as a CI/CD variable, right? Should we max out the lifetime (1 year)?